common/types/list_test.go

```diff
@@ -413,6 +427,18 @@ func TestStringList_ConvertToNative(t *testing.T) {
 	}
 }
 
+func TestStringList_ConvertToNative_ListInterface(t *testing.T) {
+	reg := NewRegistry()
+	list := NewStringList(reg, []string{"h", "e", "l", "p"})
+	val, err := list.ConvertToNative(reflect.TypeOf([]interface{}{}))
+	if err != nil {
+		t.Error("Unable to convert string list to itself.")
+	}
+	if !reflect.DeepEqual(val, []interface{}{"h", "e", "l", "p"}) {
+		t.Errorf("Got %v, expected ['h', 'e', 'l', 'p']", val)
```

Comment:
Nit: the "expected" message seems to say that you want a list of rune literals, rather than a list of strings.  Consider whether the increased clarity of the message is worth the decreased clarity of the code (with all the backslashes).